scripts/smoke-test.py

```diff
@@ -19,5 +19,5 @@
     print("smoke test passed")
     exit(0)
 else:
-    print(f"Response for {url} was: {status_code} {json}")
+    print("Response for {url} was: {status_code} {json}")
```

Comment:
removing the f might mess up the string literals `{url}` for example in the string and not print it properly.